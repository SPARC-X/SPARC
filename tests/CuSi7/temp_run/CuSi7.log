The program took 320.645 s.
