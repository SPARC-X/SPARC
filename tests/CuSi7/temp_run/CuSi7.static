***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Cu:
      0.9856277896       0.9821196020       0.9856767716
Fractional coordinates of Si:
      0.2881613217       0.2567232555       0.2601653765
      0.9134172699       0.4860040793       0.4377131907
      0.2157629973       0.7820809585       0.6977096440
      0.4388163255       0.9488096280       0.5520904873
      0.6793909734       0.3154918611       0.8311052840
      0.4456586746       0.4181411299       0.9676841197
      0.7232503333       0.7795258389       0.2833112542
Total free energy (Ha): -2.095183047003680E+02
Atomic forces (Ha/Bohr):
 -3.8092675293E-03   2.9267792755E-02   1.1514075511E-02
 -4.9530528716E-02  -4.7045125135E-02   5.2458972541E-02
  5.1522712393E-02  -3.1493791105E-02   4.1613947782E-02
 -2.6630321511E-01  -2.1825639453E-01   1.3785511043E-01
  2.6385923825E-01   2.2599263270E-01  -1.6725341933E-01
  4.6230586285E-01  -2.2941378803E-01  -2.9303459002E-01
 -4.3194332745E-01   2.3591588057E-01   2.4074699898E-01
 -2.6101474684E-02   3.5032792786E-02  -2.3901095891E-02
Stress (GPa): 
 -4.1715837471E+01   5.1147455620E+00   3.5386854966E+01 
  5.1147455620E+00  -1.5397412704E+01   5.0173108825E+00 
  3.5386854966E+01   5.0173108825E+00  -1.9173416727E+01
