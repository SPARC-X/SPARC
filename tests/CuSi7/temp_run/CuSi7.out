***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:20:39 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.26 10.26 10.26 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 103 103 103
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Gaussian
SMEARING: 0.007349864435
CHEB_DEGREE: 50
NSTATES: 32
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 9.92E-06
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: CuSi7
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 8
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 2 2
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  8
Mesh spacing                       :  0.0996117 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  CuSi7.out
Total number of atom types         :  2
Total number of atoms              :  8
Total number of electrons          :  47
Atom type 1  (valence electrons)   :  Cu 19
Pseudopotential                    :  Cu.psp8
Pseudocharge radii of atom type 1  :  6.38 6.38 6.38 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
Pseudocharge radii of atom type 2  :  6.38 6.38 6.38 (x, y, z dir)
Number of atoms of type 2          :  7
Estimated total memory usage       :  1.79 GB
Estimated memory per processor     :  229.26 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.6054729904E+01        1.329E-01        42.514
2            -2.6163371816E+01        6.324E-02        13.483
3            -2.6188652698E+01        4.998E-02        13.624
4            -2.6188058994E+01        2.104E-02        13.414
5            -2.6188678930E+01        1.155E-02        13.357
6            -2.6189095193E+01        4.932E-03        13.084
7            -2.6189452237E+01        2.924E-03        12.975
8            -2.6189636490E+01        1.742E-03        12.790
9            -2.6189725190E+01        1.656E-03        12.814
10           -2.6189761001E+01        7.228E-04        12.684
11           -2.6189776933E+01        3.567E-04        12.650
12           -2.6189783602E+01        1.852E-04        12.406
13           -2.6189786364E+01        1.451E-04        12.550
14           -2.6189787388E+01        8.207E-05        12.278
15           -2.6189787822E+01        7.796E-05        12.159
16           -2.6189787993E+01        5.262E-05        12.224
17           -2.6189788051E+01        1.456E-05        11.807
18           -2.6189788054E+01        1.115E-05        11.768
19           -2.6189788070E+01        6.133E-06        11.731
20           -2.6189788091E+01        4.768E-06        11.701
21           -2.6189788072E+01        1.251E-06        11.265
22           -2.6189788074E+01        1.521E-06        11.497
23           -2.6189788079E+01        3.984E-07        8.882
Total number of SCF: 23    

k = [0.000, 0.000, 0.000]
Occupation of state 29 = 0.000000000000000.
Occupation of state 32 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.6189788088E+01 (Ha/atom)
Total free energy                  : -2.0951830470E+02 (Ha)
Band structure energy              : -2.1531941717E+01 (Ha)
Exchange correlation energy        : -2.5488371486E+01 (Ha)
Self and correction energy         : -2.7815196429E+02 (Ha)
Entropy*kb*T                       : -1.0645950245E-02 (Ha)
Fermi level                        :  1.9352287347E-01 (Ha)
RMS force                          :  2.6734887510E-01 (Ha/Bohr)
Maximum force                      :  5.9348687257E-01 (Ha/Bohr)
Time for force calculation         :  0.396 (sec)
Pressure                           :  2.5428888967E+01 (GPa)
Maximum stress                     :  4.1715837471E+01 (GPa)
Time for stress calculation        :  1.042 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  320.645 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
